x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/data_input_flyout/steps/macros/macros_data_input.tsx

```diff
@@ -73,25 +73,25 @@ export const MacrosDataInput = React.memo<MacrosDataInputProps>(
 MacrosDataInput.displayName = 'MacrosDataInput';
 
 const END = 10 as const;
-type SubStep = 1 | 2 | 3 | typeof END;
+type SubStep = 1 | 2 | 3 | 4 | typeof END;
```

Comment:
We don't need 4 sub-steps in the macros step, right? only in the first one (rules)